src/System.Net.Sockets/src/System/Net/Sockets/UnixDomainSocketEndPoint.Unix.cs

```diff
@@ -8,7 +8,7 @@
 namespace System.Net.Sockets
 {
     /// <summary>Represents a Unix Domain Socket endpoint as a path.</summary>
-    internal sealed class UnixDomainSocketEndPoint : EndPoint
+    public sealed class UnixDomainSocketEndPoint : EndPoint
```

Comment:
@stephentoub - Do you have any thoughts on making a `public string Path {get;}` property since the constructor takes the `path` string?  Or should we leave it out until we have a requirement/scenario to expose the property?

Reply:
I'm not really aware of a need for it, though I don't have any objections to exposing it.

Reply:
OK, let's leave it out until we see a need for it.

Reply:
Similarly named pipes, mutex, semaphore etc objects all can be named but do not expose a getter for the name.